vmm/src/device_manager/legacy.rs

```diff
@@ -86,15 +106,24 @@ impl LegacyDeviceManager {
         self.io_bus
             .insert(self.stdio_serial.clone(), 0x3f8, 0x8)
             .map_err(Error::BusError)?;
-        self.io_bus
-            .insert(
-                Arc::new(Mutex::new(devices::legacy::Serial::new_sink(
-                    self.com_evt_2_4.try_clone().map_err(Error::EventFd)?,
-                ))),
-                0x2f8,
-                0x8,
-            )
-            .map_err(Error::BusError)?;
+        match self.second_serial {
+            Some(ref serial) => {
+                self.io_bus
+                    .insert(serial.clone(), 0x2f8, 0x8)
+                    .map_err(Error::BusError)?;
+            },
+            None => {
```

Comment:
Is the idea that this always creates a second serial device, but if one is not passed in, it just creates a dummy one?

Reply:
Yes.